scripts/raincloud.service.example

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Backend for the rainstorm project
+After=network.target
+
+[Service]
+User=drop
```

Comment:
how hard would it be to read the user from /config.yml instead of hard coding it?